docs/getting-started/android/running-tests.md

```diff
@@ -10,7 +10,7 @@ lane :tests do
 end
 ```
 
-Replace `"test"` with the gradle task name for running unit tests of your app.
+Replace `"test"` with the gradle task name for running unit tests in your app.
```

Comment:
This changes the meaning of the sentence in an unwanted way.

Reply:
I agree ☝️ Let's revert this, so the other change can go in.

```suggestion
Replace `"test"` with the gradle task name for running unit tests of your app.
```